js/Board.js

```diff
@@ -0,0 +1,115 @@
+class Board {
+  constructor(slots, playerBtns) {
```

Comment:
Let's have the board know its own slots (i.e. interact with DOM directly)

Let's keep players out of the board object.